sympy/stats/crv_types.py

```diff
@@ -3444,6 +3447,111 @@ def Pareto(name, xm, alpha):
 
     return rv(name, ParetoDistribution, (xm, alpha))
 
+
+#-------------------------------------------------------------------------------
+# Pert distribution ------------------------------------------------------------
+
+
+class PERTDistribution(SingleContinuousDistribution):
+    _argnames = ('a', 'b', 'c')
+
+    @property
+    def set(self):
+        return Interval(self.a, self.c)
+
+    @staticmethod
+    def check(a, b, c):
+        _value_check(b > a, "Parameter b must be greater than a.")
+        _value_check(c > b, "Parameter c must be greater than b.")
+        _value_check(c > a, "Parameter c must be greater than a.")
+
+    def pdf(self, x):
+        a, b, c = self.a, self.b, self.c
+        alpha = 1 + 4*(b - a/(c - a))
+        beta = 1 + 4*(c - b/(c - a))
+        num = (x - a)**(alpha - 1)*(c - x)**(beta - 1)
```

Comment:
```suggestion
        alpha = 1 + 4*(b - a)/(c - a)
        beta = 1 + 4*(c - b)/(c - a)
        num = (x - a)**(alpha - 1)*(c - x)**(beta - 1)
```

Reply:
I'm not sure why making this change is causing the `density(X)(z)` to fail.

Reply:
I think you need to update the tests (the ones you have added) as well.